src/docs/pages/component-docs/components/drawer/drawer-nav-group/drawer-nav-group-doc.component.ts

```diff
@@ -65,6 +85,24 @@ export class DrawerNavGroupDocComponent {
     BASIC = BASIC;
     MULTIPLE_GROUPS = MULTIPLE_GROUPS;
     SPACER = SPACER;
+    CUSTOM_CONTENT = CUSTOM_CONTENT;
+    generatedCode: string;
+
+    /* Default playground knobs */
+    knobs: NavGroupPlaygroundKnobs = {
+        title: {
+            value: 'Title',
+            type: 'string',
+            componentDefault: '',
+            hint: 'Title that appears above nav items',
```

Comment:
Fix this hint text

Reply:
I adjusted this hint text & updated the API docs too so that they match.  The previous description wasn't too accurate.